lib/sensible_logging/middlewares/tagged_logger.rb

```diff
@@ -24,6 +28,24 @@ def call(env)
 
   private
 
+  def default_options(opts)
+    {
+      logger: Logger.new(STDOUT),
+      tags: [],
+      use_default_log_tags: true,
+      tld_length: 1,
+      include_log_severity: true
+    }.merge(opts)
+  end
+
+  def setup_severity_tag(logger)
+    logger.formatter = proc do |severity, _datetime, _progname, msg|
```

Comment:
with this cause any issues if the user decides to use a logger with their own formatter? Should we account for this situation?

Reply:
They'd be able to set `include_log_severity` to `false` and then it would remain untouched.

Reply:
could we wrap it instead then?

```ruby
original_formatter = logger.formatter
logger.formatter = proc do |severity, *args|
  "[#{severity}] #{original_formatter(severity, *args)}"
end
```